integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/bigdecimal/TestBigDecimal.scala

```diff
@@ -149,8 +149,9 @@ class TestBigDecimal extends QueryTest with BeforeAndAfterAll {
   }
 
   test("test sum*10 aggregation on big decimal column with high precision") {
-    checkAnswer(sql("select sum(salary)*10 from carbonBigDecimal_2"),
-      sql("select sum(salary)*10 from hiveBigDecimal"))
+    val carbonSeq = sql("select sum(salary)*10 from carbonBigDecimal_2").collect
```

Comment:
better to change testcase as
"select cast(sum(salary)*10 as double) from carbonBigDecimal_2"

Reply:
fixed